mmdeploy/backend/tensorrt/utils.py

```diff
@@ -38,6 +38,55 @@ def load(path: str) -> trt.ICudaEngine:
         return engine
 
 
+def search_cuda_version() -> str:
+    """try import torch.version.cuda, then try use cmd.
+
+    Returns:
+        str: cuda version, for example 10.2
+    """
+
+    version = None
+    try:
+        import torch
```

Comment:
Does not prefer this way.
Here is the check env in PyTorch. We can share the same method.
https://github.com/pytorch/pytorch/blob/3ac9f6683dc8f17e030699da4df6c767f22939b6/torch/utils/collect_env.py#L347